neo/rawio/plexon2rawio/plexon2rawio.py

```diff
@@ -288,7 +301,7 @@ def _get_signal_size(self, block_index, seg_index, stream_index):
         stream_id = self.header['signal_streams'][stream_index]['id']
         stream_characteristic = list(self.signal_stream_characteristics.values())[stream_index]
         assert stream_id == stream_characteristic.id
-        return stream_characteristic.n_samples
+        return int(stream_characteristic.n_samples)  # Avoids returning a numpy.int64 scalar
```

Comment:
This change is related to this:
https://github.com/SpikeInterface/spikeinterface/pull/2223